File: java/1108 Defanging an IP Address.java
/*
1108. Defanging an IP Address
**********
Problem Link : https://leetcode.com/problems/defanging-an-ip-address/
Runtime: 0 ms
Memory Usage: 40.5 MB
**********
*/
class Solution {
    public String defangIPaddr(String address) {
        String adr=address.replace(".","[.]");
        return adr; 
    }
}

File: Python/1108. Defanging an IP Address.py
'''
1108. Defanging an IP Address
Problem Link : https://leetcode.com/problems/defanging-an-ip-address/
Runtime: 12 ms
Memory Usage: 13.6 MB
'''
class Solution(object):
    def defangIPaddr(self, address):
        return address.replace('.','[.]')